assets/scripts/build_gallery.py

```diff
@@ -163,6 +214,148 @@ def get_expectation_file_info_dict(
     return result
 
 
+def install_necessary_requirements(requirements) -> list:
+    """Install any Expectation requirements that aren't already in the venv
+
+    Return a list of things installed, so they may be uninstalled at the end
+    """
+    installed_packages = pkg_resources.working_set
+    parsed_requirements = pkg_resources.parse_requirements(requirements)
+    installed = []
+    for req in parsed_requirements:
+        is_satisfied = any(
+            [installed_pkg in req for installed_pkg in installed_packages]
+        )
+        if not is_satisfied:
+            logger.debug(f"Executing command: 'pip install \"{req}\"'")
+            status_code = execute_shell_command(f'pip install "{req}"')
+            if status_code == 0:
```

Comment:
No the traceback details end up in the gallery-tracebacks.txt file that gets shown in the pipeline. Later in the process when that Expectation attempts to run_diagnostics, it will also have a traceback for not being able to import whatever thing that failed to install.

Most things are included in the venv already via https://github.com/great-expectations/great_expectations/blob/develop/reqs/requirements-dev-all-contrib-expectations.txt